SweaterWeather/js/script.js

```diff
@@ -0,0 +1,116 @@
+var geoButton = document.getElementById('geo');
+var notifyButton = document.getElementById('notify');
+
+// Check onload status of geolocation permission.   
+navigator.permissions.query({name:'geolocation'}).then(function(result){ 
+ if (result.state == 'denied') {
+  geoButton.setAttribute('icon','communication:location-off');
+  geoButton.style.color = 'grey';
+ }
+ else if (result.state == 'granted') {
+  geoButton.setAttribute('icon','communication:location-on');
+  geoButton.style.color = 'rgb(62,81,181)';
+ }
+ else { // Prompt
+  geoButton.setAttribute('icon','communication:location-on');
+  geoButton.style.color = 'grey';
+ } 
+});
+
+// Check onload status of notifications permission.
+navigator.permissions.query({name:'notifications'}).then(function(result){ 
+ if (result.state == 'denied') {
+  notifyButton.disabled = true;
```

Comment:
WHen I count spaces, I see that "if" is indented by 1 space (instead of 2), and hten "notifyButton" is indented 2 instead of 4. The code above is also incorrect.